unit-tests-jest/numbers.test.ts

```diff
@@ -0,0 +1,81 @@
+import * as functions from './numbers';
+
+describe('evenNumbers', () => {
+  it('returns even numbers and only even numbers', () => {
+    const numbers = [2, 2.5, 4, 6, 7];
+    const result = functions.evenNumbers(numbers);
+    expect(result).toEqual([2, 4, 6]);
+  });
+  it('returns an empty array when given one', () => {
+    const numbers: number[] = [];
+    const result = functions.evenNumbers(numbers);
+    expect(result).toEqual([]);
+  });
+});
+
+describe('toDollars', () => {
+  it('returns number in dollar/cent format', () => {
+    const amount = 200.5;
+    const result = functions.toDollars(amount);
+    expect(result).toBe('$200.50');
+  });
+  it('rounds excess decimals', () => {
+    const amount = 1.1999;
+    const result = functions.toDollars(amount);
+    expect(result).toBe('$1.20');
+  });
+  it('rounds down as well as up', () => {
+    const amount = 1.2049;
+    const result = functions.toDollars(amount);
+    expect(result).toBe('$1.20');
+  });
+});
+
+describe('divideBy', () => {
+  it('does not modify the original input array', () => {
```

Comment:
Logically, you would test the happy path first, then the exceptions and side-effect behaviors like this. So I would put this one at or near the bottom of the list.